# should be casio: @test err_import
from pycasio import notcasio
